### PR TITLE
Add support for the Reporting resources

### PR DESCRIPTION
**Pull request by remi-stripe**
Adding support for the new Reporting resources. Those are in OpenAPI but not yet public but since they are starting to be used and are stable we should merge them.

It's WIP for now while I get some feedback on the implementation and the shape of the resources.

cc @stripe/api-libraries
cc @brahn-stripe

### CODE REVIEW

File: lib/resources/Reporting/ReportTypes.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+var StripeResource = require('../../StripeResource');
+
+module.exports = StripeResource.extend({
+  path: 'reporting/report_types',
+  includeBasic: ['create', 'list', 'retrieve'],
```
Comment (ob-stripe) [inline lib/resources/Reporting/ReportTypes.js]: Remove `'create'`.

**Comment by remi-stripe** [comment]
@ob-stripe Fixed, PTAL

**Comment by ob-stripe** [comment]
Released as 6.9.0.

**Comment by mbifulco** [comment]
Are there docs anywhere for reporting resources?

**Comment by ob-stripe** [comment]
@mbifulco This feature is still in private beta for now. If it sounds like something you'd be interested in, you should reach out to support at https://support.stripe.com/email to ask for more information and see if you could be invited.

**Pull request merged**